SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -5745,6 +5834,13 @@ void CEulerSolver::SetActDisk_BCThrust(CGeometry *geometry, CSolver **solver_con
   bool ActDisk_Info;
   su2double MyBCThrust, BCThrust_Init;
   su2double Vector[MAXNDIM] = {0.0};
+  int iRow, nRow, iEl;
+  su2double rad_v[50]={0.0}, dCt_v[50]={0.0}, dCp_v[50]={0.0}, dCr_v[50]={0.0},
```

Comment:
I have removed the "magic numbers": because of the fact that the dimension of these vectors depends on the actuator disk input file, I have decided to introduce a dynamic allocation, so that their dimension can change depending on the input file (which can contains different sizes if more than one actuator disk is considered).
I hope that this solution can be good enough.